### PR TITLE
Enhance to support flash two files together

### PR DESCRIPTION
**Pull request by tingai1**
The feature support 3 part files together at least before, cover the file will split into two parts which is
bigger than 4G but less than 8G
such as:
    flash super super.img.part01 super.img.part02

Tracked-On: OAM-99747
Signed-off-by: Ai, Ting <ting.a.ai@intel.com>

### CODE REVIEW

**Comment by YadongQi** [comment]
So there still file size limits (0 ~ 8G], is it possible to extend it to support size larger than 8G?

**Review by GangSecurity — APPROVED** [review]

File: installer.c [hunk 1]
```diff
@@ -594,7 +594,7 @@ static void installer_flash_cmd(INTN argc, CHAR8 **argv)
 	for (int i = 0; i <  num; i++)
 		numname[i] = NULL;
 
-	if (argc > 4) {
```
Comment (GangSecurity) [inline installer.c:597]: Needs double check the original code.  seems there is no args==3 test case.

Comment (tingai1) [reply installer.c:597]: args==3 in else for if (argc > 3)

**Review by lchen43 — CHANGES_REQUESTED** [review]
You mentioned there is no limitation of 2 parts of file. Please update comment and patch to support more than 2 part of files.

**Review by YadongQi — APPROVED** [review]

**Review by lchen43 — APPROVED** [review]

**Review by JeevakaPrabu — APPROVED** [review]

**Review by YadongQi — APPROVED** [review]

**Review by kwang13 — APPROVED** [review]

**Pull request merged**